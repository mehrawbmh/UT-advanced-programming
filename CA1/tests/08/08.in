2 12:02
Banafsh 12
7
5 Modafe'an_e_Salamat 10 Tarbiat_Modarres 4 Boostan_e_Goftogoo 3 Borj_e_Milad 8 Meydan_e_Sa'adats 5 Shahid_Dadman 10 Meydan_e_Ketab
5
4 Shahid_Navab_e_Safavi 9 Roudaki 10 Komeil 4 Beryanak 3 Helal_Ahmar
Zard 6
4
4 Shademan 5 Doctor_Habibollah 12 Osatd_Moein 10 Meidan_e_Azadi
4
5 Meidan_e_enghelab_e_Eslami 4 teatr_e_Shahr 6 Ferdowsi 8 Darvaze_Dowlat
Komeil
