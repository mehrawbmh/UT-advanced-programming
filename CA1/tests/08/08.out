Towards end of Banafsh in 3 station(s)
12:35
1478
